site/source/support/common-issues.rst

```diff
@@ -38,24 +38,31 @@ If you still cannot connect, try logging into your router (the directions for wh
 I am unable to reach Embassy via it's embassy-xxxxxxxx.local (LAN) address
 ------------------------------------------------------------------------------
 
-Make sure you have successfully followed the :ref:`LAN Setup<connecting-lan>` instructions for your device. If you still cannot connect, try all the solution listed under :ref:`initial setup <setup-troubleshoot>`.
+Make sure you have successfully followed the :ref:`LAN Setup<connecting-lan>` instructions for your device. If you still cannot connect, try all the solutions listed under :ref:`initial setup <setup-troubleshoot>`.
 
 I am unable to reach Embassy via it's xxxxxxxxxxxxxxxxxx.onion (Tor) address
 --------------------------------------------------------------------------------
 
-#. **Test** - Try connecting to your Embassy using the official :ref:`Tor Browser <tor-browser>`.
+#. **Test** - Try connecting to your Embassy using the officially preferred  :ref:`Tor Browser <tor-browser>`.
```

Comment:
This should be 'official tor browser,' meaning, from the Tor Project... but you can say preferred or recommended ahead of that